polling_stations/apps/data_importers/tests/test_base_stations_importer.py

```diff
@@ -0,0 +1,78 @@
+import os
+from unittest.mock import patch
+
+from django.contrib.gis.geos import Point
+from django.test import TestCase
+
+from councils.tests.factories import CouncilFactory
+from data_importers.base_importers import BaseStationsImporter
+from data_importers.data_types import StationSet
+
+
+class MockLogger:
```

Comment:
I think using this to then assert against in your tests is a really nice solution. 

My only other suggestion would be to use a unitest `MagicMock` object instead, which would save you from needing your own class with these methods. You could then use the [autospeccing](https://docs.python.org/3/library/unittest.mock.html#autospeccing) with your actual logger class, and then you could use some of the convenience methods like [`assert_called_once_with`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_with) to check the logger is called correctly.

But like I say, think the way you are doing works fine and is really clear when reading the tests. So this is more just me suggesting a possible other way rather than saying its better!

Reply:
OK, glad it makes sense - that was my concern. I'll have a look at the `MagicMock` for other times.